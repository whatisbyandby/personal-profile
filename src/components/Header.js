import React from "react"
import { Link } from "gatsby"
import StyledNav from "./styles/StyledNav"
import StyledBackground from "./styles/StyledBackground"
import Image from "./image"
import StyledLayer from "./styles/StyledBackgroundLayer"

const Header = props => {
  return (
    <StyledBackground>
      <StyledLayer>
        <StyledNav>
          <ul>
            <li>
              <Link to="/about">About</Link>
            </li>
            <li>
              <Link to="/portfolio">Portfolio</Link>
            </li>
            <li>
              <Link to="/contact">Contact</Link>
            </li>
          </ul>
        </StyledNav>
      </StyledLayer>
    </StyledBackground>
  )
}

export default Header
