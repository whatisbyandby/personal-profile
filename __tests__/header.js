import React from "react"
import renderer from "react-test-renderer"

import Header from "../src/components/Header"
console.log(Header)

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
