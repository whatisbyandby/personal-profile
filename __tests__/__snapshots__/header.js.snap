// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="sc-AxirZ leJUON"
>
  <nav
    className="sc-AxjAm lcvwrE"
  >
    <h3>
      <a
        href="/"
      >
        Scott Perkins
      </a>
    </h3>
    <ul>
      <li
        id="about-nav"
      >
        <a
          href="/about"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="/portfolio"
        >
          Portfolio
        </a>
      </li>
      <li>
        <a
          href="/contact"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
